packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -503,6 +499,186 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
             }
           });
         });
+
+        describe('NON-BASIC Tiers', () => {
+          const createAliasForNonBasicPlans = async (mode: SubscriptionTier) => {
+            const aliasAccount = await Utils.createAliasAccount(
+              mirrorNode,
+              global.accounts[0],
+              requestId,
+              initialBalance,
+            );
+
+            const hbarSpendingPlan = await hbarSpendingPlanRepository.create(mode, requestDetails, mockTTL);
+
+            await ethAddressSpendingPlanRepository.save(
+              { ethAddress: aliasAccount.address, planId: hbarSpendingPlan.id },
+              requestDetails,
+              mockTTL,
+            );
+
+            return { aliasAccount, hbarSpendingPlan };
+          };
+
+          describe('Pre-configured spending plans', async () => {
+            const expectedNonBasicPlans = {
+              PRIVILEGED_ALPHA: {
+                id: 'c758c095-342c-4607-9db5-867d7e90ab9d',
+                name: 'PRIVILEGED_ALPHA',
+                ethAddresses: ['0x7d102fe71af42790fe31b126c1f49766376ca2b5'],
+                ipAddresses: ['127.0.0.1'],
+                subscriptionTier: 'PRIVILEGED',
+              },
+              PRIVILEGED_BETA: {
+                id: 'a68488b0-6f7d-44a0-87c1-774ad64615f2',
+                name: 'PRIVILEGED_BETA',
+                ethAddresses: ['0x40183ec818c1826114767391989ff2eaebc2b91e'],
```

Comment:
We should have a test that verifies the same PRIVILEGED plan can be shared by multiple addresses

Reply:
Good point! Removed and reused contents from the JSON file.

Reply:
hmm good point added more addresses to the config file